tests/unit/src/main/scala/tests/BaseWorksheetLspSuite.scala

```diff
@@ -8,10 +8,19 @@ import scala.meta.internal.metals.UserConfiguration
 import scala.meta.internal.metals.clients.language.MetalsSlowTaskResult
 import scala.meta.internal.metals.{BuildInfo => V}
 
-abstract class BaseWorksheetLspSuite(scalaVersion: String)
-    extends BaseLspSuite("worksheet") {
-  override def initializationOptions: Option[InitializationOptions] =
-    Some(InitializationOptions.Default.copy(decorationProvider = Some(true)))
+abstract class BaseWorksheetLspSuite(
+    scalaVersion: String,
+    useVirtualDocuments: Boolean,
+    suiteNameSuffix: String
+) extends BaseLspSuite(s"worksheet-$suiteNameSuffix") {
+
+  override protected def initializationOptions: Option[InitializationOptions] =
```

Comment:
We could make `initializationOptions` a var and change `useVirtualDocuments` for a single case instead.